src/modules/cmdpal/extensionsdk/Microsoft.CmdPal.Extensions.Helpers/ListHelpers.cs

```diff
@@ -44,6 +44,17 @@ public static IEnumerable<IListItem> FilterList(IEnumerable<IListItem> items, st
             .Select(score => score.ListItem);
     }
 
+    public static IEnumerable<T> FilterList<T>(IEnumerable<T> items, string query, Func<string, T, int> scoreFunction)
```

Comment:
This is cool. I haven't seen much of this in C# yet where we pass Func in as method param. neat.